CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
```

Comment:
We typically use the GH releases to note the changes in each release. Also, you can mark them as a "pre-release" which marks it as `alpha` (sort of).